src/app/screens/LNURLPay.tsx

```diff
@@ -50,9 +52,9 @@ function LNURLPay(props: Props) {
       // Get the invoice
       const params = {
         amount: valueMSat, // user specified sum in MilliSatoshi
+        comment: comment, // an optional parameter to pass the LN WALLET user's comment to LN SERVICE. Note on comment length: GET URL's accept around ~2000 characters for the entire request string. Therefore comment can only be as large as to fit in the URL alongisde any/all of the properties outlined above.*
```

Comment:
You can use an object shorthand here: https://alligator.io/js/object-property-shorthand-es6/